client-libs/python/openpipe/shared.py

```diff
@@ -5,66 +5,62 @@
 from openpipe.api_client.models.report_json_body_tags import (
     ReportJsonBodyTags,
 )
+
+from .api_client_fern.client import OpenPipeApi, AsyncOpenPipeApi
+
 from openai.types.chat import ChatCompletion
 import os
 import pkg_resources
 import json
 from typing import Any, Dict, List, Union
 
 
-def configure_openpipe_client(openpipe_options={}) -> AuthenticatedClient:
-    configured_client = AuthenticatedClient(
-        base_url="https://app.openpipe.ai/api/v1",
-        token="",
-        raise_on_unexpected_status=True,
-    )
-
+def configure_openpipe_client(
+    client: Union[OpenPipeApi, AsyncOpenPipeApi], openpipe_options={}
+):
     if os.environ.get("OPENPIPE_API_KEY"):
-        configured_client.token = os.environ["OPENPIPE_API_KEY"]
+        client._client_wrapper._token = os.environ["OPENPIPE_API_KEY"]
 
     if os.environ.get("OPENPIPE_BASE_URL"):
-        configured_client._base_url = os.environ["OPENPIPE_BASE_URL"]
+        client._client_wrapper._base_url = os.environ["OPENPIPE_BASE_URL"]
 
-    if openpipe_options and "verify_ssl" in openpipe_options:
-        configured_client._verify_ssl = bool(openpipe_options["verify_ssl"])
+    # if openpipe_options and "verify_ssl" in openpipe_options:
```

Comment:
I'm not sure what the equivalent of `verify_ssl` is with Fern. We should figure that out.

Reply:
We might not need this, I think it was just because when running against localhost the client would complain that SSL wasn't configured. If Fern isn't complaining about that might be fine.